src/NexusMods.App.UI/Pages/LoadoutGrid/Columns/ModEnabled/ModEnabledViewModel.cs

```diff
@@ -2,6 +2,7 @@
 using System.Reactive.Disposables;
 using System.Reactive.Linq;
 using System.Windows.Input;
+using ExCSS;
```

Comment:
```suggestion
```

Common mistake, `ExCSS` also contains a `Unit` type but we want the one from `System.Reactive`.